distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Postgres/Postgres_Dialect.enso

```diff
@@ -596,18 +596,18 @@ replace args metadata =
                     _ ->
                         SQL_Builder.code "REGEXP_REPLACE(" ++ input ++ ", " ++ escaped_pattern ++ ", " ++ replacement ++ ")"
         Regex ->
-            pattern_string = SQL_Builder.interpolation raw_pattern.pattern_string
+            pattern = SQL_Builder.interpolation raw_pattern.pattern
```

Comment:
tbh I actually preferred the old way here.
This is internals - we don't need to care about user-facing naming here.

The `pattern_string` makes it clear this is a string/text value. The `pattern` is unclear - is it a string or is it a `Regex.Pattern` instance, or something?

If you want to avoid _string_ even in internals code, I'd at least rename this variable to `pattern_text`, to make it clear that this is the text representation of the pattern.